### DISCUSSION TITLE
Unable to establish ssl connection

### DISCUSSION THREAD

**Discussion by rebaz94** [planetscale/discussion#90] [Bug Reports] [answered]
Hi.
I created a new database and want to connect using SSL but its not connected and in the popup **ssl not enabled** 
here is configuration that I used

![Screen Shot 2021-08-25 at 2 40 23 PM](https://user-images.githubusercontent.com/11982812/130784508-9b473513-d9f9-4b3d-8788-922d150cd56e.png)

and 

![Screen Shot 2021-08-25 at 2 40 29 PM](https://user-images.githubusercontent.com/11982812/130784563-c09962dc-e4c3-4393-89a1-85e51c690021.png)

I tried many configuration but not working but in terminal i can connect using this command
```
mysql -h fuv0x8.us-east-1.psdb.cloud -u user -ppscale --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem
```

please could you guide me what is the problem and how to fix it? 

Thanks

**Comment by jgreet** [comment, marked as answer]
Not entirely sure what's going on with MySQL Workbench, but here's what I've learned so far in my own testing.

Using the same settings shown in your screenshots, I can connect from Workbench 8.0.25 on a Mac running OSX 11.5.2 but *only* with the password saved in the keychain. With the password cleared, the connection will fail with no password prompt ever appearing.

With those same settings, **Test Connection** will show a successful connection with `SSL: not enabled`. However, I don't think that can be true. PlanetScale won't allow unencrypted connections. You can verify that by flipping the **Use SSL** dropdown to **No** and trying **Test Connection** again.

We'll keep looking at this in case we've introduced some kind of incompatibility on our end. In the meantime, you could also connect Workbench via the `pscale connect` feature in our [command-line client](https://github.com/planetscale/cli).

https://docs.planetscale.com/tutorials/connect-any-application#connect-using-client-certificates

**Comment by rebaz94** [comment]
Thanks for your information, I detected same thing as you said and tried ```pscale connect``` its work great.
But I don't understand with the same configuration why is not connecting in  the **Sequel Pro** and it return error 
```
MySQL said: SSL connection error: unknown error number
```
![Screen Shot 2021-08-25 at 8 12 43 PM](https://user-images.githubusercontent.com/11982812/130836911-ea5e103e-28b5-439d-9efb-9619e9b939b4.png)

![Screen Shot 2021-08-25 at 8 12 50 PM](https://user-images.githubusercontent.com/11982812/130836925-ed1c0977-9467-4a75-afa5-2ad30a96e957.png)

mainly I use Sequel Pro but now I can not connect to it. So can you tell me any solution, please?

**Comment by jgreet** [comment]
I don't think Sequel Pro is going to work. Could you give the updated fork Sequel Ace a try? I was able to connect it to PlanetScale with no issues.

https://github.com/Sequel-Ace/Sequel-Ace

Sequel Pro hasn't been updated in a while and it has several outstanding SSL problems: https://github.com/sequelpro/sequelpro/issues/2854

**Reply by rebaz94** [reply]
Yes you right, the problem is Sequel Pro, now its work using Sequel-Ace with SSL enabled.

Thanks so much for your help
